src/project/internal/notationproject.cpp

```diff
@@ -641,10 +655,39 @@ Ret NotationProject::doSave(const muse::io::path_t& path, engraving::MscIoMode i
                 LOGW() << ret.toString();
             }
         } else {
-            Ret ret = fileSystem()->move(savePath, targetContainerPath, true);
-            if (!ret) {
-                return ret;
+            Ret ret = muse::make_ok();
+
+            if (!targetContainerPath.contains(" - ALL_ZEROS_CORRUPTED_ORIG_ONLY.mscz")) {
```

Comment:
I don't quite get the purpose of this code. We already have a way (which is safer as it is not always enabled, but only in the dev mode) to corrupt the score for testing purposes. Let's keep the logic as simple as possible